tools/__tasks__/compile/hash/index.js

```diff
@@ -12,26 +12,51 @@ const mkdirpp = pify(mkdirp);
 
 const { hash, target } = require('../../config').paths;
 
-function getSrcHash(fileMapping, file) {
-    const fileHash = hasha.fromFileSync(path.resolve(target, file), { algorithm: 'md5' });
-    return Object.assign(fileMapping, {
-        [file]: path.join(path.dirname(file), fileHash, path.basename(file)),
-    });
-}
+const standardAssetsMap = () => {
+    const assets = glob.sync('**/!(*.map|*webpack*.js)', { nodir: true, cwd: target });
+    return assets.reduce((assetMap, asset) => {
+        const assetHash = hasha.fromFileSync(path.resolve(target, asset), { algorithm: 'md5' });
+        return Object.assign(assetMap, {
```

Comment:
What do you guys think about this rule: https://github.com/airbnb/javascript#whitespace--after-blocks

Personally I like it. I find code much easier to read with nice whitespace. I also read [maintainable JS](http://dl.finebook.ir/book/4a/13598.pdf) which I found really useful.